backend/api/v2beta1/python_http_client/kfp_server_api_v2beta1/__init__.py

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+
+# flake8: noqa
+
+"""
+    Kubeflow Pipelines API
+
+    This file contains REST API specification for Kubeflow Pipelines. The file is autogenerated from the swagger definition.
+
+    Contact: kubeflow-pipelines@google.com
+    Generated by: https://openapi-generator.tech
+"""
+
+
+from __future__ import absolute_import
+
+__version__ = "2.0.0-alpha.6"
+
+# import apis into sdk package
+from kfp_server_api_v2beta1.api.experiment_service_api import ExperimentServiceApi
+
+# import ApiClient
+from kfp_server_api_v2beta1.api_client import ApiClient
```

Comment:
Good catch. I missed that one in the previous PR. The package name should remain the same.

Reply:
I wonder who are the main users of this package? From what I can tell it's only KFP SDK. Now that we have two sets of API, we will generate two Python packages. There might be a way to combine them into one package, but I'm not sure.

Reply:
We will not need to generate two Python packages, nor do we need to combine them into one. v1 and v2 APIs would live in the same package under different versions. SDK would be the switching point for using v1 vs. v2: To use v1 API, users use SDK 1.*, to use v2 API, users use SDK 2.*. 
That being said, we need to redirect v1 API to the new path so that it won't be a breaking change for existing users.

Reply:
I see. I have regenerated the python clients so there is no api version in the package names.